gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -207,6 +209,145 @@ def __repr__(self):
         [str(s) for s in [self.name, self.genotype, self.phaseset, self.info]])
 
 
+class _VcfSink(filebasedsink.FileBasedSink):
+  """A sink for writing VCF files."""
+
+  def __init__(self,
+               file_path,
+               num_shards=1,
+               coder=coders.ToStringCoder(),
+               compression_type=CompressionTypes.UNCOMPRESSED,
+               headers=None):
+    """Initialize a _TextSink.
+
+    Args:
+      file_path: The file path to write to. The files written will begin
+        with this prefix, followed by a shard identifier (see num_shards), and
+        end in a common extension, if given by file_name_suffix. In most cases,
+        only this argument is specified and num_shards, shard_name_template, and
+        file_name_suffix use default values.
+      num_shards: The number of files (shards) used for output. If not set, the
+        service will decide on the optimal number of shards.
+        Constraining the number of shards is likely to reduce
+        the performance of a pipeline.  Setting this value is not recommended
+        unless you require a specific number of output files.
+      coder: Coder used to encode each line.
+      compression_type: Used to handle compressed output files. Typical value
+        for VCF files is CompressionTypes.UNCOMPRESSED. If set to
+        CompressionTypes.AUTO, file_path's extension will be used to detect
+        compression.
+      headers: A list of VCF meta-information lines describing the INFO,
+        FILTER, and FORMAT entries in each record and a header line describing
```

Comment:
Hmm aren't INFO and FORMAT just required to read the .vcf file back into beam? It might be a bit confusing to include information essential for us to read the file in the sink.

Reply:
I see. Good point. I'd still prefer to not include FILTER in this comment as it's a special case and totally optional.

Reply:
Good point. Fixed it!